src/doc/gui_manual/WorkingWithFiles/Supported_File_Types.rst

```diff
@@ -3,32 +3,171 @@
+
+Here is an example ``.visit`` file that groups time steps together. These files should contain 
+1 time step per file.
+
+.. code-block:: none
+
+   timestep0.silo
+   timestep1.silo
+   timestep2.silo
+   timestep3.silo
+   ...
+
+Here is an example ``.visit`` file that groups various smaller domain files into a whole dataset 
+that VisIt can visualize. Note the use of the ``!NBLOCKS`` directive and how it designates the 
+number of files in a time step that constitute the whole domain. The ``!NBLOCKS`` directive must 
+be on the first line of the file. In this example, we have 2 time steps each composed of 4 domain 
+files.
+
+.. code-block:: none
+   :emphasize-lines: 1
+
+   !NBLOCKS 4
+   timestep0_domain0.silo
+   timestep0_domain1.silo
+   timestep0_domain2.silo
+   timestep0_domain3.silo
+   timestep1_domain0.silo
+   timestep1_domain1.silo
+   timestep1_domain2.silo
+   timestep1_domain3.silo
+   ...
+
+You may also explicitly indicate the *time* associated with a file (or group of block files)
+using the ``!TIME`` directive like so...
+
+.. code-block:: none
+   :emphasize-lines: 1,2,7
+
+   !NBLOCKS 4
+   !TIME 1.01
+   timestep0_domain0.silo
+   timestep0_domain1.silo
+   timestep0_domain2.silo
+   timestep0_domain3.silo
+   !TIME 2.02
+   timestep1_domain0.silo
+   timestep1_domain1.silo
+   timestep1_domain2.silo
+   timestep1_domain3.silo
+   ...
+
+
+Point3D Files
+"""""""""""""
+
+Point3D files are four or fewer columns of ASCII values with some header text to indicate the
+variable names associated with each column and a ``coordflag`` entry to indicate how to 
+interpret the columns of data as coordinates. Point3D files can be used to define discrete
+points in 1, 2 and 3 dimensions having a single scalar value associated with each point.
+Some examples are below. The Point3D file...
+
+.. code-block:: none
+
+    x y z value
+    0 0 0 0
+    0 0 1 1
+    0 1 0 2
+    0 1 1 3
+    1 0 0 4
+    1 0 1 5
+    1 1 0 6
+    1 1 1 7
+
+Defines a collection of 8 points in 3 dimensions have a scalar variable named *value*.
+Below, the ``#coordflag`` directive is used to define the same collection of 8 points in
+3 dimensions as the previous example except where the columns holding the z-coordinate
+and the scalar variable are interleaved.
+
+.. code-block:: none
+   :emphasize-lines: 2
+
+    x y value z
+    #coordflag xyvz
+    0 0 0 0
+    0 0 1 1
+    0 1 2 0
+    0 1 3 1
+    1 0 4 0
+    1 0 5 1
+    1 1 6 0
+    1 1 7 1
+
+In the example below, the ``#coordflag`` directive is used to a collection of 3 points in
+*2 dimensions* where each point has a velocity magnitude value associated with it.
+
+.. code-block:: none
+   :emphasize-lines: 2
+
+    x y velocity
+    #coordflag xyv
+    0 0 1
+    0 1 1.01
+    1 0 2.02
+
+There are some
+`additional examples <https://www.visitusers.org/index.php?title=Reading_point_data#Using_Point3D_files>`_
+of Point3D files on the VisIt_ wiki pages.
```

Comment:
An example of coordflag zv would be useful.

Reply:
Well, its an `xv` example and I added it.